src/Middlewares/PermissionMiddleware.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Orderonlineid\Permission\Middlewares;
+
+use Closure;
+use Exception;
+
+/**
+ * Class PermissionMiddleware
+ * @package Orderonlineid\Permission\Middlewares
+ */
+class PermissionMiddleware
+{
+	/**
+	 * @param $request
+	 * @param Closure $next
+	 * @param $permission
+	 *
+	 * @return mixed
+	 * @throws Exception
+	 */
+	public function handle($request, Closure $next, $permission): mixed
```

Comment:
i think it better to use variadic function for the permission,
so we doesn't have to explode the permission

`public function handle($request, Closure $next, ...$permissions): mixed`